bottlerocket/testsys/Cargo.toml

```diff
@@ -18,7 +18,7 @@ maplit = "1"
 model = { version = "0.0.2", path = "../../model" }
 serde = "1.0.145"
 serde_plain = "1"
-serde_json = "1.0.86"
+serde_json = "1.0.87"
```

Comment:
I don't think we should be doing this. It increases the chances of a build breaking when a different crate has some reason to require, e.g. `<=1.0.86`. I think that we should use `1.0` and let cargo do its thing, only bumping versions in the lock file. (Alternatively we could leave this as `1.0.86` and only change the lock file, but I'm not sure if dependabot has a feature for that.)

The comment is general. `serde` is very stable, but other crates can be problematic if we absolutely insist that we must have the latest patch version in our Cargo.toml file.

Reply:
That seems reasonable @webern. Want to propose a patch to change that? Then we can just drop this PR.

Reply:
Yeah, I have a PR open that will cause a rebase in this one if we merge it... here #631